### PR TITLE
Version 0.8.0.5:  Include package-level documentation

### PR DESCRIPTION
**Pull request by lhsego**
Completed this:

- add entries to NEWS.md about what you have done
- bump the version to 0.8.0.5 in both NEWS.md and DESCRIPTION and the package help file
- update the date in DESCRIPTION
- add a dummy .R file with roxygen for trelliscope, with the version, date, and a link to http://tessera.io/docs-trelliscope/
- re-run roxygen and commit the updated .Rd files

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+FEATURES / CHANGES
+
+- Improved various aspects of the documentation of `makeDisplay()`
+- Added overall package documentation.  See `help(trelliscope)`
+
+BUG FIXES
+
+- fix `makeDisplay()` to correctly launch trelliscope views on Windows
+
 Version 0.8.0.3
```
Comment (hafen) [inline NEWS.md]: Can you merge your features and bug fixes in with what's in the 0.8.0.3 section?  In other words, merge the version 0.8.0.5 section in to 0.8.0.3 and keep the header of Version 0.8.0.5.

File: R/trelliscope-package.R [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+##'
+##' \tabular{ll}{
+##' Package: \tab trelliscope\cr
+##' Type: \tab Package\cr
+##' Version: \tab 0.8.0.5\cr
+##' Date: \tab 2015-01-29\cr
+##' License: \tab BSD_3_clause + file LICENSE\cr 
+##' Maintainer: \tab Ryan Hafen \email{rhafen@@gmail.com}\cr}
+##' 
+##  Landon here:  I'm displaying the full link in the text so that people will know where they
```
Comment (hafen) [inline R/trelliscope-package.R]: Full link is good.  Can you remove these two comment lines?

**Comment by hafen** [comment]
Thanks!  Made a couple small comments.

File: R/trelliscope-package.R (hunk 2 above)
Comment (lhsego) [inline R/trelliscope-package.R]: done

File: NEWS.md (hunk 1 above)
Comment (lhsego) [inline NEWS.md]: Done

**Comment by hafen** [comment]
Merged. Thanks!

**Pull request merged**